part-spec/externalFile.json

```diff
@@ -0,0 +1,40 @@
+"$id": "https://github.com/edatasheets/edatasheets.github.io/blob/main/part-spec/externalFile.json",
+"$schema": "http://json-schema.org/draft-07/schema#",
+"title": "Specification for referencing an external file",
+"package": {
+    "type": "object",
+    "properties": {
+        "fileDescription": {
+            "description": "text description of the contents of an external file",
+            "type": "string"
+        },
+        "fileType": {
+            "description": "type of file being linked",
+            "examples": [
+                "brd",
+                "ibis"
```

Comment:
We don't want to give the impression  that externalFile can only be used for package information. So it may be good to add something like json on the type file - This would be handy if we have to reference the component file from the orderableMPN file like we discussed in the WG.

Reply:
good thinking